### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
Just some cleanup I thought I would do while trolling about the code base.

### CODE REVIEW

File: dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java [hunk 1]
```diff
@@ -16,7 +16,10 @@
 
 package com.google.cloud.tools.opensource.dashboard;
 
-import freemarker.core.ParseException;
+import com.google.cloud.tools.opensource.dependencies.Artifacts;
```
Comment (elharo) [inline dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: This looks like internal import order. I don't think our externally published Eclipse stylesheet matches this yet. If so, I'd prefer not to have to manually reorder our imports.

Comment (garrettjonesgoogle) [reply dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: I used the google-java-format plugin in IntelliJ to produce this order (so it's easily reproducible exclusively in OSS). I can revert this reordering if it disrupts your workflow though.

Comment (elharo) [reply dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: We may want to take an action item or even an OKR to update the corresponding Eclipse plugin, but until that's done please revert.

**Comment by garrettjonesgoogle** [comment]
Feedback addressed, PTAL @elharo

**Pull request merged**